### PR TITLE
Devcon namespace changes for devon4j

### PR DESCRIPTION
**Pull request by Rohit90700**
this PR is for issue #141

### CODE REVIEW

**Comment by Rohit90700** [comment]
Hi @vapadwal ,
i have removed all oasp4j  references from java classes.

File: CONTRIBUTING_GUIDE.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ toc::[]
 
 === Notes on Code Contributions
 
-Both projects, devon and OASP, are intended to be easy to contribute to. One service allowing such simplicity is GitHub was therefore selected as preferred collaboration platform.
+Both projects, devon and DEVON, are intended to be easy to contribute to. One service allowing such simplicity is GitHub was therefore selected as preferred collaboration platform.
```
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: devon and DEVON is a nonsense. Only use devonfw. We sent the new branding strategy some days ago. Please take a look to it and fix this.

File: CONTRIBUTING_GUIDE.adoc [hunk 2]
```diff
@@ -23,11 +23,11 @@ It is mandatory to follow the <<Contributor Covenant Code of Conduct,code of con
 
 Git is a version control system used for a coordinated and versioned collaboration of computer files. It enables a project to be easily worked on by multiple developers and contributors.
 
-GitHub is an online repository used by deonvfw and OASP in order to host the corresponding files. Using the command line tool or the GUI Tool "GitHub Desktop" a user can easily manage project files. There are private and public repositories. Public ones (like OASP) can be accessed by everyone, private repositories (like devon) require access permissions.
+GitHub is an online repository used by deonvfw and DEVON in order to host the corresponding files. Using the command line tool or the GUI Tool "GitHub Desktop" a user can easily manage project files. There are private and public repositories. Public ones (like DEVON) can be accessed by everyone, private repositories (like devon) require access permissions.
```
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: The same as above

File: CONTRIBUTING_GUIDE.adoc [hunk 3]
```diff
@@ -23,11 +23,11 @@ It is mandatory to follow the <<Contributor Covenant Code of Conduct,code of con
...
 
 ==== Creating a new user account
 
-The devon and OASP projects use GitHub as hosting service. Therefore you'll need an account to allow collaboration. Visit https://github.com/join?source=header-home[this page] to create a new account. If available, use *your CORP username* as GitHub username and *your CORP email address*.
+The devon and DEVON projects use GitHub as hosting service. Therefore you'll need an account to allow collaboration. Visit https://github.com/join?source=header-home[this page] to create a new account. If available, use *your CORP username* as GitHub username and *your CORP email address*.
```
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: The same as above

File: CONTRIBUTING_GUIDE.adoc [hunk 4]
```diff
@@ -37,12 +37,12 @@ An in-depth documentation on basic Git syntax and usage can be found on the http
 
 === Structure of our projects
 
-In total, there are three GitHub projects regarding OASP and devon:
+In total, there are three GitHub projects regarding DEVON and devon:
```
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: The same as above.

File: CONTRIBUTING_GUIDE.adoc [hunk 5]
```diff
@@ -37,12 +37,12 @@ An in-depth documentation on basic Git syntax and usage can be found on the http
...
 
-* link:https://github.com/oasp-forge[*oasp-forge*]
+* link:https://github.com/devon-forge[*devon-forge*]
```
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: For the moment, we leave this, but this repository does not exist.

File: CONTRIBUTING_GUIDE.adoc [hunk 6]
```diff
@@ -37,12 +37,12 @@ An in-depth documentation on basic Git syntax and usage can be found on the http
...
 +
 Repository used for work on the guide - Similar to the according devon repository link:https://github.com/devonfw/devon-guide/wiki[*devon-guide*]
-* link:https://github.com/oasp/[*oasp*]
+* link:https://github.com/devon/[*devon*]
```
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: The link should start with https://github.com/devonfw

Comment (sjimenez77) [reply CONTRIBUTING_GUIDE.adoc]: Ok, it is related with the next change. This OASP references should be removed as they will not exist anymore.

File: CONTRIBUTING_GUIDE.adoc [hunk 7]
```diff
@@ -55,17 +55,17 @@ This branch contains software in release state.
...
 ** *master*
 
 === Contributing to our projects
 
-In order to contribute to our projects, developers must follow the following <<Development Guidelines,development guidelines>>. Other sources about contributing to devon/OASP:
+In order to contribute to our projects, developers must follow the following <<Development Guidelines,development guidelines>>. Other sources about contributing to devon/DEVON:
 
-* https://github.com/oasp/oasp4j/wiki/oasp-code-contributions[OASP code contributions]
-* https://github.com/oasp/oasp4j/wiki/oasp-documentation[OASP documentation]
+* https://github.com/devon/devon4j/wiki/devon-code-contributions[DEVON code contributions]
```
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: https://github.com/devonfw/devon4j

File: CODE_OF_CONDUCT.adoc [hunk 8]
```diff
@@ -9,7 +9,7 @@ toc::[]
 :source-highlighter: rouge
 :listing-caption: Listing
 
-IMPORTANT: This document is the *Official Covenant Code of Conduct* that must be present in every OASP or Devonfw project at the root folder as `CODE_OF_CONDUCT.asciidoc` or `CODE_OF_CONDUCT.md`. Please, include this contents in your repository and  the *Product Owner email address* in the right place below. 
+IMPORTANT: This document is the *Official Covenant Code of Conduct* that must be present in every DEVON or Devonfw project at the root folder as `CODE_OF_CONDUCT.asciidoc` or `CODE_OF_CONDUCT.md`. Please, include this contents in your repository and  the *Product Owner email address* in the right place below.
```
Comment (sjimenez77) [inline CODE_OF_CONDUCT.adoc]: Only "in every devonfw project"

File: CONTRIBUTING_GUIDE.adoc (hunk 1 above)
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: devon and DEVON? Only we have devonfw now.

File: CONTRIBUTING_GUIDE.adoc (hunk 2 above)
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: The same as above.

File: CONTRIBUTING_GUIDE.adoc (hunk 3 above)
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: The same as above.

File: CONTRIBUTING_GUIDE.adoc (hunk 5 above)
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: This repository does not exist

File: CONTRIBUTING_GUIDE.adoc (hunk 4 above)
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: The same as above.

File: CONTRIBUTING_GUIDE.adoc (hunk 6 above)
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: https://github.com/devonfw

File: CONTRIBUTING_GUIDE.adoc [hunk 9]
```diff
@@ -55,17 +55,17 @@ This branch contains software in release state.
 
 * link:https://github.com/devonfw/[*devonfw*]
 +
-This is a private repository. You have to be logged in and have permissions to access the project and its repositories. Similar to OASP, there are usually two branches:
+This is a private repository. You have to be logged in and have permissions to access the project and its repositories. Similar to DEVON, there are usually two branches:
```
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: This is not private anymore.

File: CONTRIBUTING_GUIDE.adoc (hunk 7 above)
Comment (sjimenez77) [inline CONTRIBUTING_GUIDE.adoc]: https://github.com/devonfw/devon4j

Comment (sjimenez77) [reply CONTRIBUTING_GUIDE.adoc]: As this, review please all the changes.

**Comment by AbhayChandel** [comment]
This PR points to devon4j-3.0.0 SNAPSHOT template which is unreleased hence not available in maven central repository. Kindly do not merge this PR to base repository as it will break the code. Once devon4j-3.0.0 is released, _SNAPSHOT_ should be bumped from the devon4j template version in this PR, test this PR and then merged with the base repository.

**Comment by vapadwal** [comment]
As devon4j 3.0.0 is released and Conflicts are resolved , therefore merging this PR.
Also as mentioned above devon4j run is working

**Pull request merged**